File: bitget-java-sdk-api/src/main/java/com/bitget/openapi/dto/request/spot/SpotOrderInfoReq.java
package com.bitget.openapi.dto.request.spot;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;

/**
 * @Author: bitget-sdk-team
 * @Date: 2022-09-30 10:46
 * @DES: Get order details request
 */
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class SpotOrderInfoReq implements Serializable {

    private static final long serialVersionUID = -1L;

    /**
     * Currency pair
     */
    private String symbol;
    /**
     * Order Id
     */
    private String orderId;
    /**
     * Client Order Id
     */
    private String clientOrderId;
}

File: bitget-node-sdk-api/src/lib/model/spot/order/SpotOrderInfoReq.ts
export class SpotOrderInfoReq {
    /**
     * Currency pair
     */
    private _symbol!:string;
    /**
     * Order Id
     */
    private _orderId!:string;
    /**
     * Client Order Id
     */
    private _clientOrderId!:string;


    get symbol(): string {
        return this._symbol;
    }

    set symbol(value: string) {
        this._symbol = value;
    }

    get orderId(): string {
        return this._orderId;
    }

    set orderId(value: string) {
        this._orderId = value;
    }

    get clientOrderId(): string {
        return this._clientOrderId;
    }

    set clientOrderId(value: string) {
        this._clientOrderId = value;
    }
}